apps/condo/domains/user/oidc/OIDCBearerTokenKeystonePatch.js

```diff
@@ -0,0 +1,88 @@
+const { get } = require('lodash')
+
+const { getSchemaCtx } = require('@core/keystone/schema')
+
+const adapter = require('./adapter')
+
+function getOidcToken (req) {
+    try {
+        let authHeader = req.headers.authorization || req.headers.Authorization
+        if (!authHeader) {
+            return
+        }
+
+        if (Array.isArray(authHeader)) {
+            authHeader = authHeader[0]
+        }
+
+        const [type, token] = authHeader.split(' ')
+
+        if (type !== 'Bearer' || token.length !== 43 || token.includes('.')) {
```

Comment:
error will be thrown if token === undefined (in case split return single part)

Reply:
That's true %) Unfortunately, I took this logic from the Keystone core and it can't be changed there